packages/ember/lib/index.js

```diff
@@ -439,6 +440,11 @@ Ember.TextField = TextField;
 Ember.TextArea = TextArea;
 Ember.LinkComponent = LinkComponent;
 
+Object.defineProperty(Ember, 'setComponentManager', {
```

Comment:
darn, I'm sorry for the run around here (I know you implemented exactly what I said in the last comment), I think this still needs to be underscored

Can you change it to be `_setComponentManager`?

Reply:
Haha, I did `setComponentManager` first, then backed off. Shoulda gone with my gut.